subseas_flex_fcst/widgets.py

```diff
@@ -0,0 +1,113 @@
+from dash import dcc
```

Comment:
a couple of 

#bs_size="sm",
className="my-1",

changed into

size="sm", className="m-1 d-inline-block w-auto"

but I would not be able to tell why...

Reply:
There seems to be differences in Sentence on the handling of groups, but since I am not using Sentence here... that might have come from another piece of work...

Reply:
Sorry... I am mixing up files when looking at diff. My best guess is that I did nothing to widget here other than they way to import things when used in pyCPT, that I had to change back when moving back to python-maprooms. I am not sure... but my bet is that it could just use the widgets.py in onset_maprooms...

is there a way to see the history of commits for 1 file here:
https://github.com/kjhall-iri/pycpt-remic/pull/7/files
?

Reply:
Not exactly what you asked for, but
https://github.com/kjhall-iri/pycpt-remic/blame/a61764aa9a3c8d4b5e98996518afe3d56bff44cc/src/maproom/widgets.py

The only change that was made after the initial commit is commenting out `bs_size="sm"`

Forget about it for now, we'll harmonize the various widget.py's in a separate PR.